src/Microsoft.DotNet.Configurer/LocalizableStrings.resx

```diff
@@ -126,13 +126,14 @@ Telemetry
 --------------
 The .NET Core tools collect usage data in order to improve your experience. The data is anonymous and does not include command-line arguments. The data is collected by Microsoft and shared with the community.
 You can opt out of telemetry by setting a DOTNET_CLI_TELEMETRY_OPTOUT environment variable to 1 using your favorite shell.
-You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.
-
-Configuring...
+You can read more about .NET Core tools telemetry @ https://aka.ms/dotnet-cli-telemetry.</value>
```

Comment:
@nguerrera @livarcocc I'm guessing that splitting the message like this is going to be problematic from a loc point of view, since we've likely lock'd the resource strings at this point?  Are there additional changes that need to happen to deal with the xlf stuff regardless?